### ISSUE TITLE
Loss ending up being None too much

### ISSUE THREAD

**Issue by Reknez** [ongakken/ACARIS#2]
```
mdl output: {'logits': tensor([[  3.8244,   1.2785, -12.5631],
        [  0.8900,   2.8808, -11.3245],
        [ -1.1820,   1.0067,  -9.8143],
        [  5.6538,   5.5505, -13.1790],
        [  5.0708,  -0.5134, -11.8855],
        [  5.1792,  -1.4311, -11.2753],
        [  5.2483,   2.5216, -12.8563],
        [  1.2075,  -5.4756,  -9.6146],
        [  1.5299,   2.3952, -11.5845],
        [  6.3338,   3.1505, -12.9497],
        [  3.3177,  -2.9943, -10.6362],
        [  6.7617,  -0.0713, -11.4039],
        [  5.1446,  -1.2685, -11.4186],
        [  0.5130,   2.6556, -10.9190],
        [  3.3887,   1.9819, -12.2857],
        [  4.0987,   0.9359, -12.4902]], device='cuda:0')}
labels before loss: tensor([0, 1, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 1, 0], device='cuda:0')
loss: None
```

Module output looks correct. Labels also look correct, but loss is still being `None`. That leaves only `userEmbs`. There's something wrong with either their creation or their passing to `forward()`.
